src/caja-navigation-window.c

```diff
@@ -1397,6 +1425,24 @@ caja_navigation_window_split_view_on (CajaNavigationWindow *window)
         caja_navigation_window_pane_hide_location_bar (pane, TRUE);
     }
     G_GNUC_END_IGNORE_DEPRECATIONS;
+
+
+    paned = GTK_WIDGET (window->details->split_view_hpane);
+    scale = gtk_widget_get_scale_factor (GTK_WIDGET (window->sidebar));
+    window->details->extra_pane_width =
+        g_settings_get_int (caja_window_state, CAJA_WINDOW_STATE_EXTRA_PANE_POSITION) * scale;
+    if (window->details->extra_pane_width <= 0)
+        window->details->extra_pane_width = gtk_widget_get_allocated_width (paned) / 2;
+    gtk_paned_set_position (GTK_PANED (paned), window->details->extra_pane_width);
```

Comment:
before setting the panel position you have to check its validity, ie use `CLAMP (x, low, high)`